doc/produs_unterlagen/materials/java-cheat-sheet.md

```diff
@@ -0,0 +1,147 @@
+Hier ist dein Cheat Sheet in einer übersichtlichen und formatierten Version – ideal zum Ausdrucken oder Teilen mit Studierenden:
+## 🧠 Java Cheat Sheet – Grundlagen
+
+### 🔢 Variablen
+
+```java
+int i = 5;                 // Ganze Zahl
+float f = 3.5f;            // Kleine Fließkommazahl (mit 'f')
+double d = 3.12545123;     // Große Fließkommazahl
+char c = 'c';              // Einzelnes Zeichen
+String s = "Das ist Text"; // Zeichenkette
+```
+
+**Rechnen und Verkettung:**
+
+```java
+int i = 5;
+int v = i * 2; 
+i = 3;
+String s = "i ist jetzt " + i;
+```
+
+---
```

Comment:
willst du wirklich überall mit einer _horizontal line_ trennen? in der gh-preview gibt es pro überschrift ja sowieso eine trennzeile (automatisch). oder ist das für eine konvertierung nach word oder so gedacht? aber auch da stört das irgendwie.